hack/cluster-deploy.sh

```diff
@@ -87,6 +87,6 @@ until _kubectl -n kubevirt get kv kubevirt; do
 done
 
 # wait until KubeVirt is ready
-_kubectl wait -n kubevirt kv kubevirt --for condition=Ready --timeout 6m || (echo "KubeVirt not ready in time" && exit 1)
```

Comment:
hm, why it can take 12m? all images should be under the container registry, so we should not have latency problems.

Reply:
Re 12m, I kept the timeout large. This https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/2395/pull-kubevirt-e2e-kind-k8s-sriov-1.14.2/1155957792633262080 is a run that failed for that reason, missing the 6m deadline.

It happened a couple of times here and there so I am not sure I can debug it. Maybe an alternative would be to print out the content of kubevirt namespace in case of failure?

Reply:
yep it can be useful you can also to print virt-operator log

Reply:
ok, I'll close this and open another PR.

Reply:
can you reproduce this long time locally? I'm wondering if there is something going wrong with the kind cluster.

Reply:
Nope. For sure, I needed to relax some timeouts so maybe it's just that those machine where we have sriov cards are just less powerful.
Another thing I can do is to time it in this throw away PR and see how long it takes compared to the other providers / machines.